src/acceleration/BaseQNAcceleration.cpp

```diff
@@ -218,15 +213,24 @@ void BaseQNAcceleration::performAcceleration(
 
   profiling::Event e("cpl.computeQuasiNewtonUpdate", profiling::Synchronize);
 
+  // If its the first iteration of the first time window we want to save the timegrid that the quasi-Newton method uses.
```

Comment:
```suggestion
  // We can only initialize here since we need to know the time grid already.
```
?
(don't describe the what, but the why)